packages/adapters/src/kubernetes-instance-adapter.ts

```diff
@@ -84,6 +84,10 @@ IComponent {
             INSTANCE_ID: instanceId,
         }).map(([name, value]) => ({ name, value }));
 
+        const runnerImage = config.engines.node
```

Comment:
Other instance adapters actually have a reversed logic. They switch to python3 if it's present in the engines config. I think this behavior should resemble that to avoid situation in which sequence without an engine specified will default to Python (I guess we want Node to be the default)